docs/developer/architecture/core/logging-service-configuration.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[logging-service-configuration-example]]
+== Logging Service Configuration Example
+
+As with any configuration in the {kib} platform, logging configuration is validated against the predefined schema and if there are
+any issues with it, {kib} will fail to start with the detailed error message.
+
+Once the code acquired a logger instance it should not care about any runtime changes in the configuration that may
+happen: all changes will be applied to existing logger instances under the hood.
```

Comment:
It seems to be an implementation detail. Not sure we need to provide it in the user-facing documentation. WDYT?

Reply:
++ Good point, probably not necessary